applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -688,19 +693,21 @@ static void audio_datapath_i2s_start(void)
 	int ret;
 
 	/* Double buffer I2S */
-	uint8_t *tx_buf_one;
-	uint8_t *tx_buf_two;
-	uint32_t *rx_buf_one;
-	uint32_t *rx_buf_two;
+	uint8_t *tx_buf_one = NULL;
+	uint8_t *tx_buf_two = NULL;
+	uint32_t *rx_buf_one = NULL;
+	uint32_t *rx_buf_two = NULL;
 
 	/* TX */
+#if CONFIG_STREAM_BIDIRECTIONAL || (CONDFIG_AUDIO_DEV == HEADSET)
```

Comment:
Should have parenthesis around here:
`` #if (CONFIG_STREAM_BIDIRECTIONAL || (CONDFIG_AUDIO_DEV == HEADSET))``

This applies to all #if statements in this PR